ui/src/org.ts

```diff
@@ -1,22 +1,54 @@
-import * as api from "./api"
-import { createValidationStore } from "./validation"
-import { Transaction } from './transaction';
-import { EmojiAvatar } from "./avatar"
+import { createValidationStore } from "./validation";
+import { EmojiAvatar } from "./avatar";
+import { Transaction } from "./transaction";
+
+import * as api from "./api";
+import * as event from "./event";
+import * as remote from "./remote";
+
+// TYPES.
+export interface Org {
+  name: string;
+  avatarFallback: EmojiAvatar;
+}
+
+interface Fetch extends event.Event<Kind> {
+  kind: Kind.Fetch;
+  id: string;
+}
+
+interface RegisterOrgInput {
+  id: string;
+}
+
+type Msg = Fetch;
 
 export enum RegistrationFlowState {
   NameSelection,
   TransactionConfirmation
 }
 
-export interface Org {
-  name: string;
-  avatarFallback: EmojiAvatar;
+// STATE
```

Comment:
Usually we order: types, state, event, helpers in our other files.

Reply:
What is out of order here?

Reply:
The Msg, Kind and Fetch are all part of the event system, so is update. The state should just move directly after the types and then all the event stuff.

Reply:
Like so: https://github.com/radicle-dev/radicle-upstream/pull/345/commits/75034f7cf4fd4a19a2b26f96f8acaf6c19c24982?

I guess the `// TYPES` section threw me off, I just stuck everything that looked like a type in there.